### ISSUE TITLE
WellBet App Logo ändern & Verbindungsfehler mit Datenbank checken

### ISSUE THREAD

**Issue by diplomod** [diplomod/Geonova-new-#12]
Das habe ich ganz vernachlässigt. Muss unbedingt rein. Geonova investiert hier jetzt, bis wir wieder einen Sponsor haben.

Source Code: https://www.dropbox.com/sh/2algtipmvmiac1s/AADUO8e2m0NXKq8lcvvN0kL5a?dl=0 

Philip Bayr vermutet den Fehler auf Seite der Website.
Das bitte nochmal checken.

**Comment by philipdomann** [comment]
Android und iPhone haben den Fehler?

**Comment by diplomod** [comment]
Ja. iPhone fix und beim Adnroid hatte ich das gleiche - hat immer irgend ein Problem mit Verbindung zur Datenbank.
Aber bitte prüft ihr Android nochmal.

**Comment by diplomod** [comment]
Wir haben auch einen neuen Sponsor. Kann man das Logo und einen Banner austauschen sowie die Texte entsprechend ändern?


![img_2331 2](https://cloud.githubusercontent.com/assets/7487883/24699766/621cfb38-19f5-11e7-9d37-2a6e786fe876.PNG)
![img_2332](https://cloud.githubusercontent.com/assets/7487883/24699768/622111d2-19f5-11e7-984c-bc9b1056ded4.PNG)
![img_2333 2](https://cloud.githubusercontent.com/assets/7487883/24699765/621c99c2-19f5-11e7-8f76-d2db684a721b.PNG)
![img_2334](https://cloud.githubusercontent.com/assets/7487883/24699767/62208c80-19f5-11e7-99dc-291ff1278922.PNG)
